### ISSUE TITLE
Disabling Addons

### ISSUE THREAD

**Issue by accell187** [Healix/Gw2Launcher#411]
Running on Windows 11
Basic Mode
Localized execution disabled
Localized CEF is on
Rename CefHost.exe is on
-----------------------------
Lately, most of the time when keeping arcdps's d3d11.dll in the game folder causes various problems, from Firefox crashing, to accounts not loading completely and sequentially blocking other accounts from loading (gw2.exe remains running in the background)
-----------------------------
Currently manually removing or renaming d3d11.dll from the game folder each time. 
Is it possible to disable addons from Launcher settings?

**Comment by Healix** [comment]
No, there's no option to toggle addons, but you could create a copy of the account and use localized execution to exclude or only include addons on it.

GW2 remaining in the background has been a problem since January. It seems to only happen when using arcdps on Windows 11. It should only affect Gw2Launcher when updating, which requires you to manually end GW2's process to get it to continue.

If you just want to toggle d3d11.dll for an account, you can use a batch script to rename it back and forth. Save this as a .bat file in the same folder as d3d11.dll and run it to toggle it.

if exist d3d11.dll (ren d3d11.dll d3d11.dll.x) else (ren d3d11.dll.x d3d11.dll)

You could do a similar thing as localized execution by having an account with a script to run before launch that toggles it on and another account that toggles it off.
